src/pages/library/DataBrowserDetails.js

```diff
@@ -103,7 +107,7 @@ const MainContent = ({ snapshot }) => {
   ])
 }
 
-const Sidebar = ({ snapshot }) => {
+const Sidebar = ({ snapshot, setShowRequestAccessModal }) => {
```

Comment:
If you move the state into this component you can remove the `setShowRequestAccessModal` prop.